src/components/BorrowerProfile/LendCta.vue

```diff
@@ -60,25 +66,67 @@
 								${{ priceOption }}
 							</option>
 						</kv-ui-select>
+
 						<!-- Lend button -->
 						<kv-ui-button
-							v-if="!showAdding && !inBasket"
+							v-if="lendButtonVisibility"
 							class="tw-inline-flex tw-flex-1"
 							@click="addToBasket"
+							v-kv-track-event="[
+								'Lending',
+								'Add to basket',
+								'Lend now'
```

Comment:
You could use the `ctaButtonText` variable here instead of `'Lend now'` so that the tracking event label always matches the actual text displayed on the button.

Reply:
Good idea!

Reply:
Good idea indeed, change complete.